pkg/instancegroups/instancegroups.go

```diff
@@ -169,6 +197,13 @@ func (r *RollingUpdateInstanceGroup) RollingUpdate(rollingUpdateData *RollingUpd
 
 				glog.Warningf("Cluster validation failed after removing instance, proceeding since fail-on-validate is set to false: %v", err)
 			}
+			if rollingUpdateData.Interactive {
+				var stop_prompting bool = PromptInteractive(nodeName)
```

Comment:
you should be able to do

`stopPrompting, err := PromptInteractive(nodeName)`

Test if err is nil, and if not return it up the stack.

Reply:
Implemented in this manner.